`npm test` failing on clean checkout

OS: macOS 10.12.4
Node version: 7.9.0
NPM version: 4.5.0

```shell
$ git clone https://github.com/mobxjs/mobx-react.git
...
$ npm install
...
$ npm test
> mobx-react@4.1.8 test:console /Users/bryan/code/mobx-react
> browserify -x react-native -x react/addons -x react/lib/ReactContext -x react/lib/ExecutionEnvironment test/*.js -t [ babelify --presets [ es2015 react ] --plugins [ transform-decorators-legacy transform-class-properties ] ] | tape-run | tap-spec && npm run test:ts

(node:30758) DeprecationWarning: Calling an asynchronous function without callback is deprecated.
Error: Cannot find module 'react-test-renderer/shallow' from '/Users/bryan/code/mobx-react/node_modules/enzyme/build'
    at /Users/bryan/code/mobx-react/node_modules/browser-resolve/node_modules/resolve/lib/async.js:46:17
    at process (/Users/bryan/code/mobx-react/node_modules/browser-resolve/node_modules/resolve/lib/async.js:173:43)
    at ondir (/Users/bryan/code/mobx-react/node_modules/browser-resolve/node_modules/resolve/lib/async.js:188:17)
    at load (/Users/bryan/code/mobx-react/node_modules/browser-resolve/node_modules/resolve/lib/async.js:69:43)
    at onex (/Users/bryan/code/mobx-react/node_modules/browser-resolve/node_modules/resolve/lib/async.js:92:31)
    at /Users/bryan/code/mobx-react/node_modules/browser-resolve/node_modules/resolve/lib/async.js:22:47
    at FSReqWrap.oncomplete (fs.js:114:15)
internal/streams/legacy.js:59
      throw er; // Unhandled stream error in pipe.
      ^

Error: javascript required
    at Stream.<anonymous> (/Users/bryan/code/mobx-react/node_modules/browser-run/index.js:35:34)
    at _end (/Users/bryan/code/mobx-react/node_modules/through/index.js:65:9)
    at Stream.stream.end (/Users/bryan/code/mobx-react/node_modules/through/index.js:74:5)
    at Stream.method [as end] (/Users/bryan/code/mobx-react/node_modules/duplexer/index.js:47:39)
    at Stream.<anonymous> (/Users/bryan/code/mobx-react/node_modules/throughout/index.js:7:25)
    at _end (/Users/bryan/code/mobx-react/node_modules/through/index.js:65:9)
    at Stream.stream.end (/Users/bryan/code/mobx-react/node_modules/through/index.js:74:5)
    at Stream.onend (internal/streams/legacy.js:44:10)
    at emitNone (events.js:91:20)
    at Stream.emit (events.js:188:7)
```

I'm not sure where to start looking to get this fixed, hopefully somebody can point me in the right direction.

--- a/package.json
+++ b/package.json
@@ -46,6 +46,7 @@
     "react": "^15.5.3",
     "react-addons-test-utils": "^15.5.1",
     "react-dom": "^15.5.3",
+    "react-test-renderer": "^15.5.4",
     "tap-spec": "^4.1.1",
     "tape": "^4.2.2",
     "tape-run": "2.1.0",

